app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -13,6 +13,15 @@ import { INPUT_DELAY } from "discourse-common/config/environment";
 import discourseLater from "discourse-common/lib/later";
 import domUtils from "discourse-common/utils/dom-utils";
 
+let disabledBindings = [];
+export function disableDefaultKeyboardShortcuts(bindings) {
+  disabledBindings = disabledBindings.concat(bindings);
+}
+
+export function clearDisabledDefaultKeyboardBindings() {
+  disabledBindings = [];
+}
+
```

Comment:
```suggestion
const disabledBindings = [];
export function disableDefaultKeyboardShortcuts(bindings) {
  disabledBindings.push(bindings);
}

export function clearDisabledDefaultKeyboardBindings() {
  disabledBindings.length = 0;
}

```

100% code-golf–you can ignore if you want...

Reply:
Tested the change, this won't work though :) I did not dig into why.